growattServer/__init__.py

```diff
@@ -1,1173 +1,8 @@
-        result = response.json()
-        
-        if not result.get('success', False):
-            raise Exception(f"Failed to update TLX inverter time segment: {result.get('msg', 'Unknown error')}")
-        
-        return result
-
-    def update_tlx_inverter_setting(self, serial_number, setting_type, parameter):
-        """
-        Alias for setting parameters on a tlx hybrid inverter
-        See README for known working settings
-
-        Arguments:
-        serial_number -- Serial number (device_sn) of the inverter (str)
-        setting_type -- Setting to be configured (str)
-        parameter -- Parameter(s) to be sent to the system (str, dict, list of str)
-                (array which will be converted to a dictionary)
-
-        Returns:
-        JSON response from the server whether the configuration was successful
-        """
-        default_parameters = {
-            'op': 'tlxSet',
-            'serialNum': serial_number,
-            'type': setting_type
-        }
-
-        # If parameter is a single value, convert it to a dictionary
-        if not isinstance(parameter, (dict, list)):
-            parameter = {'param1': parameter}
-        elif isinstance(parameter, list):
-            parameter = {f'param{index+1}': param for index, param in enumerate(parameter)}
-
-        return self.update_inverter_setting(serial_number, setting_type, 
-                                            default_parameters, parameter)
-
-
-    def update_noah_settings(self, serial_number, setting_type, parameters):
-        """
-        Applies settings for specified noah device based on serial number
-        See README for known working settings
-
-        Arguments:
-        serial_number -- Serial number (device_sn) of the noah (str)
-        setting_type -- Setting to be configured (str)
-        parameters -- Parameters to be sent to the system (dict or list of str)
-                (array which will be converted to a dictionary)
-
-        Returns:
-        JSON response from the server whether the configuration was successful
-        """
-        default_parameters = {
-            'serialNum': serial_number,
-            'type': setting_type
-        }
-        settings_parameters = parameters
-        
-        #If we've been passed an array then convert it into a dictionary
-        if isinstance(parameters, list):
-            settings_parameters = {}
-            for index, param in enumerate(parameters, start=1):
-                settings_parameters['param' + str(index)] = param
-        
-        settings_parameters = {**default_parameters, **settings_parameters}
-
-        response = self.session.post(self.get_url('noahDeviceApi/noah/set'), 
-                                     data=settings_parameters)
-        
-        return response.json()
-
 
+# Import everything from base_api to ensure backward compatibility
+from .base_api import *
+# Import the V1 API class
+from .v1_api import GrowattApiV1
```

Comment:
I think we can rename this to OpenApiV1 as they seem to refer to it being that in their doc..